content/developers/overview/managing-deploy-keys.md

```diff
@@ -83,6 +83,32 @@ See [our guide on Git automation with tokens][git-automation].
 7. Select **Allow write access** if you want this key to have write access to the repository. A deploy key with write access lets a deployment push to the repository.
 8. Click **Add key**.
 
+##### Using multiple repositories on one server
+
+If you use multiple repositories on one server, you will need to generate a dedicated key pair for each one. You can't reuse a deploy key for multiple repositories.
+
+In the server's SSH configuration file (usually `~/.ssh/config`), add an alias entry for each repository. For example:
+
+```bash
+Host github.com-repo-0
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-0_deploy_key
+
+Host github.com-repo-1
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-1_deploy_key
+```
+
+* `Host github.com-repo-0` - The repository's alias.
+* `Hostname github.com` - Configures the alias to use the {% data variables.product.product_name %} server.
+* `IdentityFile=/home/user/.ssh/repo-0_deploy_key` - Assigns a private key to the alias.
+
+With these entries added, you can then use the alias to clone a repository, and the unique deploy key is automatically presented. For example:
+
+```bash
+$ git clone git@github.com-repo-1:github-user/repo-1.git
+```
```

Comment:
@martin389 Because this article can also apply to GHES, could we condition all mentions of `github.com` depending on the page version?

e.g. for free-pro-team: `github.com`
enterprise-server/ae: `my-GHE-server.com` (or something 🙃)

Reply:
I've now rephrased the explanation of the `Hostname` entry to just refer to the hostname that the user wants to alias, which should clarify that this is a configurable setting that can also accept a GHES hostname 🤔

Reply:
@martin389 yep, that description is great generic description for that, but I think we should have something else instead of `github.com` in the examples for GHES-versioned pages.

You should be able to replace all the examples with something like:

`{% if currentVersion == "free-pro-team@latest" %}github.com{% else %}my-GHE-hostname.com{% endif %}`

Reply:
Added this is now, thanks @lucascosti!